dallinger/command_line/__init__.py

```diff
@@ -509,10 +506,17 @@ def hibernate(app):
         heroku_app.addon_destroy(addon)
 
 
-def _current_hits(service, app):
-    if app is not None:
-        return service.get_hits(hit_filter=lambda h: h.get("annotation") == app)
-    return service.get_hits()
+def get_recruiter(recruiter_name):
+    """Return a recruiter by name."""
+    available_recruiters = ["mturk", "prolific"]
+    if recruiter_name == "mturk":
+        return MTurkRecruiter(is_dummy=True)
```

Comment:
Maybe it would be clearer if instead of making the argument `is_dummy` we called it `validate`?

Reply:
Good suggestion, I renamed it to `not_validate_config`, since the default is to validate the config.